src/components/item-display-panel/item-display-panel.component.ts

```diff
@@ -0,0 +1,75 @@
+import { Component, Directive, Input, SimpleChange } from '@angular/core';
+import { ModalModule } from 'ngx-bootstrap/modal';
+import { Observable } from 'rxjs/Observable';
+
+@Component({
+    selector: 'ux-item-display-panel',
+    templateUrl: './item-display-panel.component.html',
+    exportAs: 'ux-item-display-panel',
+    host: {
+        '(document:click)': 'clickOff($event)',
+        '(document:keyup.escape)': 'hide()'
+    }
+})
+export class ItemDisplayPanelComponent { 
+    @Input() top: number;
+    @Input() shadow: boolean;
+    @Input() visible: boolean;
```

Comment:
Make visible a two way binding as allowing them to know when it is open could be useful